metadata_test.go

```diff
@@ -0,0 +1,500 @@
+				ContainSubstring("Package: libsasl2-dev"),
+				ContainSubstring("Package: libsasl2-modules"),
+				ContainSubstring("Package: libsasl2-modules-gssapi-mit"),
+				ContainSubstring("Package: libselinux1"),
+				ContainSubstring("Package: libselinux1-dev"),
+				ContainSubstring("Package: libsigc++-2.0-0v5"),
+				ContainSubstring("Package: libsigc++-2.0-dev"),
+				ContainSubstring("Package: libsigsegv2"),
+				ContainSubstring("Package: libsqlite0"),
+				ContainSubstring("Package: libsqlite0-dev"),
+				ContainSubstring("Package: libsqlite3-0"),
+				ContainSubstring("Package: libsqlite3-dev"),
+				ContainSubstring("Package: libssl-dev"),
+				ContainSubstring("Package: libsysfs-dev"),
+				ContainSubstring("Package: libsysfs2"),
+				ContainSubstring("Package: libtasn1-6"),
+				ContainSubstring("Package: libtasn1-6-dev"),
+				ContainSubstring("Package: libterm-ui-perl"),
+				ContainSubstring("Package: libtiffxx5"),
+				ContainSubstring("Package: libtirpc-dev"),
+				ContainSubstring("Package: libtool"),
+				ContainSubstring("Package: libunwind8"),
+				ContainSubstring("Package: libustr-1.0-1"),
+				ContainSubstring("Package: libustr-dev"),
+				ContainSubstring("Package: libwmf0.2-7"),
+				ContainSubstring("Package: libwrap0"),
+				ContainSubstring("Package: libwrap0-dev"),
+				ContainSubstring("Package: libxapian-dev"),
+				ContainSubstring("Package: libxapian30"),
+				ContainSubstring("Package: libxdot4"),
+				ContainSubstring("Package: libxslt1-dev"),
+				ContainSubstring("Package: libxslt1.1"),
+				ContainSubstring("Package: libyaml-0-2"),
+				ContainSubstring("Package: libyaml-dev"),
+				ContainSubstring("Package: lockfile-progs"),
+				ContainSubstring("Package: lsof"),
+				ContainSubstring("Package: lzma"),
+				ContainSubstring("Package: mercurial"),
+				ContainSubstring("Package: net-tools"),
+				ContainSubstring("Package: ocaml-base-nox"),
+				ContainSubstring("Package: openssh-client"),
+				ContainSubstring("Package: openssl1.0"),
+				ContainSubstring("Package: psmisc"),
+				ContainSubstring("Package: python"),
+				ContainSubstring("Package: python-bzrlib"),
+				ContainSubstring("Package: rsync"),
+				ContainSubstring("Package: ruby"),
+				ContainSubstring("Package: subversion"),
+				ContainSubstring("Package: sysstat"),
+				ContainSubstring("Package: ttf-dejavu-core"),
+				ContainSubstring("Package: ubuntu-minimal"),
+				ContainSubstring("Package: unixodbc"),
+				ContainSubstring("Package: unixodbc-dev"),
+				ContainSubstring("Package: unzip"),
+				ContainSubstring("Package: uuid"),
+				ContainSubstring("Package: uuid-dev"),
+				ContainSubstring("Package: wget"),
+				ContainSubstring("Package: zip"),
+			)))
+
+			// Virtual packages
+			Expect(image).To(HaveFileWithContent("/var/lib/dpkg/status", SatisfyAll(
+				ContainSubstring("Provides: gnupg1-curl"),
+				ContainSubstring("Provides: libreadline6-dev"),
+				ContainSubstring("Provides: libunwind8-dev"),
+			)))
```

Comment:
nitpick but can these all go in the same test expectation as the non-virtual packages?

Reply:
+1 - I don't think we need to separate these packages out.